### ISSUE TITLE
zcoin payments

### ISSUE THREAD

**Issue by RiskingTime** [MarkdaleMGMT/transaction-methods#26]

**Comment by BeingJustifiedFreely** [comment]
Yes! I am running a desktop client, but not the most recent version with the protocol change, the one prior to still running Zerocoin. I have some minted Zcoin that I minted using the special minting function that I have not yet spent on that wallet. Is there anything I can do? I have been looking at the debug options but am not very clear on the direction. Any thoughts?
